library/ibm-semeru-runtimes

```diff
@@ -5,167 +5,139 @@ GitRepo: https://github.com/ibmruntimes/semeru-containers.git
 GitFetch: refs/heads/ibm
 
 #-----------------------------openj9 v8 images---------------------------------
-Tags: open-8u422-b05-jdk-focal, open-8-jdk-focal
+Tags: open-jdk8u432-b06-jdk-focal, open-8-jdk-focal
 Architectures: amd64, ppc64le, s390x, arm64v8
-GitCommit: 8236ca97febf5f9934e97e2d6a1ecbfaa87fa610
+GitCommit: 0d581de9acf0fefbd9f7c5d7844fa0838d49a27d
 Directory: 8/jdk/ubuntu/focal
 File: Dockerfile.open.releases.full
 
-Tags: open-8u422-b05-jdk-jammy, open-8-jdk-jammy
-SharedTags: open-8u422-b05-jdk, open-8-jdk
+Tags: open-jdk8u432-b06-jdk-jammy, open-8-jdk-jammy
+SharedTags: open-jdk8u432-b06-jdk, open-8-jdk
 Architectures: amd64, ppc64le, s390x, arm64v8
-GitCommit: 8236ca97febf5f9934e97e2d6a1ecbfaa87fa610
+GitCommit: 0d581de9acf0fefbd9f7c5d7844fa0838d49a27d
 Directory: 8/jdk/ubuntu/jammy
 File: Dockerfile.open.releases.full
 
-Tags: open-8u422-b05-jre-focal, open-8-jre-focal
+Tags: open-jdk8u432-b06-jre-focal, open-8-jre-focal
```

Comment:
`open-8u422-b05-*` -> `open-jdk8u432-b06-*`
Is this tag name change on purpose (the extra `jdk`)? It seems a bit out of place. It is on each of the JDK/JRE 8 tags sets.

Reply:
Thanks @yosifkit for pointing out the change in tag format. I have corrected them. Thanks!!